drivers/flash/Kconfig.emulator

```diff
@@ -0,0 +1,133 @@
+# Kconfig - Flash driver emulator config
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if FLASH && FLASH_EMULATOR
+
+config FLASH_EMULATOR_BASE_ADDR
+	hex
+	prompt "Base address of the flash"
+	default 0x00
+	depends on FLASH_EMULATOR
```

Comment:
This is already inside an `if FLASH_EMULATOR` block, so `depends on FLASH_EMULATOR` is redundant.

`if` works like syntactic sugar for adding a `depends on` to everything in the block.